app/src/main/res/layout/fragment_neighbours.xml

```diff
@@ -1,10 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent" android:layout_height="match_parent">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/activity_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center_horizontal"
+    android:background="@color/white">
+
 
     <TextView
+        android:id="@+id/manage_exercises_title"
```

Comment:
nit name in camel case

Reply:
thanks, I missed this. Will change it in the following pull request :)